### PR TITLE
Update python client header

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

**Comment by saicheems** [comment]
@geigerj 
@bjwatson 
@tbetbetbe

File: vgen/src/main/resources/io/gapi/vgen/py/main.snip [hunk 1]
```diff
@@ -239,14 +239,14 @@
             client_config = pkg_resources.resource_string(
                 __name__, '{@jsonBaseName}_client_config.json')
             {@constructDefaults(service)}
-            google_apis_agent = '{}/{};{};gax/{};python/{}'.format(
+            goog_api_client = '{}/{};{};gax/{};python/{}'.format(
```
Comment (geigerj) [inline vgen/src/main/resources/io/gapi/vgen/py/main.snip]: Should this be space- or semicolon-separated? I've seen spaces elsewhere (e.g., Ruby)....

Comment (saicheems) [inline vgen/src/main/resources/io/gapi/vgen/py/main.snip]: I... don't know
I'll send an email

Comment (saicheems) [inline vgen/src/main/resources/io/gapi/vgen/py/main.snip]: Changed it to spaces based on the thread

**Comment by bjwatson** [comment]
@saicheems Do you know why the build is breaking?

**Comment by saicheems** [comment]
Forgot to update the baseline, should succeed after my push just now

**Comment by bjwatson** [comment]
Cool, thanks.

**Comment by geigerj** [comment]
LGTM

**Pull request merged**